diff --git a/linux-tkg-config/prepare b/linux-tkg-config/prepare
index d7a8ed3..846c9d5 100644
--- a/linux-tkg-config/prepare
+++ b/linux-tkg-config/prepare
@@ -157,8 +157,9 @@ _tkg_initscript() {
 
   if [ -z "$_OPTIPROFILE" ] && [ ! -e "$_path"/cpuschedset ]; then
     # Prompt about optimized configurations. Available variable values will overwrite customization.cfg/external config ones.
-    plain "Do you want to use a predefined optimized profile?"
-    read -rp "`echo $'  > 1.Custom\n    2.Ryzen Desktop (Performance)\n    3.Other Desktop (Performance)\nchoice[1-3?]: '`" _OPTIPROFILE;
+    plain "Using custom profile (_OPTIPROFILE=1) by default"
+    _OPTIPROFILE=1
+    #read -rp "`echo $'  > 1.Custom\n    2.Ryzen Desktop (Performance)\n    3.Other Desktop (Performance)\nchoice[1-3?]: '`" _OPTIPROFILE;
   fi
   if [ "$_OPTIPROFILE" = "2" ]; then
     source "$_where"/linux-tkg-config/ryzen-desktop-profile.cfg && msg2 "Ryzen Desktop (Performance) profile will be used." && msg2 ""
@@ -361,7 +362,7 @@ _tkg_srcprep() {
   fi
 
   # graysky's cpu opts - https://github.com/graysky2/kernel_gcc_patch
-  
+
   msg2 "Applying graysky's cpu opts patch"
   if [ "${_distro}" = "Arch" ]; then
     tkgpatch="$srcdir/more-uarches-for-gcc-v10-and-kernel-${opt_ver}.patch" && _tkg_patcher
@@ -647,7 +648,7 @@ CONFIG_DEBUG_INFO_BTF_MODULES=y\n
 
   if [ "$_processor_opt" = "native" ]; then
     echo "CONFIG_MNATIVE=y" >> ./.config
-  fi  
+  fi
 
   _cpu_marchs=("k8" "k8sse3" "k10" "barcelona" "bobcat" "jaguar" "bulldozer" "piledriver")
   _cpu_marchs+=("steamroller" "excavator" "zen" "zen2" "mpsc" "atom" "core2" "nehalem" "westmere")
@@ -663,7 +664,7 @@ CONFIG_DEBUG_INFO_BTF_MODULES=y\n
     else
       echo "# CONFIG_M${_march_upper} is not set" >> ./.config
     fi
-  done  
+  done
 
   # Disable some debugging
   if [ "${_debugdisable}" = "true" ]; then
