diff --git a/PKGBUILD b/PKGBUILD
index 2fed235..397179c 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -8,6 +8,8 @@ pkgdesc="zlib replacement with optimizations for 'next generation' systems"
 arch=('i686' 'x86_64')
 url="https://github.com/zlib-ng/zlib-ng"
 license=('custom:zlib')
+provides=('zlib')
+conflicts=('zlib')
 depends=('glibc')
 makedepends=('cmake')
 options=('staticlibs')
@@ -24,6 +26,7 @@ build() {
     -DCMAKE_INSTALL_PREFIX="/usr" \
     -DCMAKE_INSTALL_LIBDIR="lib" \
     -DWITH_GTEST=OFF \
+    -DZLIB_COMPAT=ON
     ./
   make -C "_build"
 }
